hybridse/src/udf/default_udf_library.cc

```diff
@@ -666,9 +666,37 @@ void DefaultUdfLibrary::InitStringUdf() {
                 select date_format(date(1590115420000),"%Y-%m-%d");
                 --output "2020-05-22"
             @endcode)");
+    RegisterExternal("upper")
+        .args<StringRef, StringRef>(
+            static_cast<void (*)(codec::StringRef*, codec::StringRef*, bool*)>(udf::v1::upper))
+        .return_by_arg(true)
+        .doc(R"(
+            @brief Convert all the characters to uppercase.
+
+            Example:
+
+            @code{.sql}
+                SELECT UPPER('Sql') as str1;
+                --output "SQL"
+            @endcode
+            @since 0.4.0)");
+    RegisterExternal("ucase")
```

Comment:
RegisterAlias("substr", "substring");
I am sorry, I miss `RegisterAlias` usage in the guide

Reply:
RegisterAlias("ucase", "upper");